tests/test_pr_intent_inherit.sh

```
@@ -32,11 +32,27 @@ SKILL="${SCRIPT_DIR}/skills/pr/SKILL.md"
-assert_file_contains "PR opened/edited/synchronize/reopened トリガー" "$WF" "opened, edited, synchronize, reopened"
+assert_file_contains "PR opened/edited/synchronize/reopened/ready_for_review トリガー" "$WF" "opened, edited, synchronize, reopened, ready_for_review"
 assert_file_contains "Fork PR 除外"              "$WF" "head.repo.full_name == github.repository"
 assert_file_contains "draft 除外"                "$WF" "!github.event.pull_request.draft"
 assert_file_contains "permissions: pull-requests: write" "$WF" "pull-requests: write"
 assert_file_contains "permissions: issues: write"  "$WF" "issues: write"
+# gh コマンドの --repo "$REPO" 明示は引数順序に依存しない形で検査（formatting 変更耐性）
+# 該当 gh コマンドを含む行を抽出し、その行に --repo "$REPO" が含まれることを確認
+assert_line_has_repo_flag() {
+  local desc="$1"
+  local cmd_pattern="$2"
+  if grep -E -- "$cmd_pattern" "$WF" 2>/dev/null | grep -q -F -- '--repo "$REPO"'; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)